src/rpc/server/scala/build.sbt

```diff
@@ -250,7 +250,7 @@ lazy val serv = project
     bashScriptExtraDefines += bashExtras,
     batScriptExtraDefines += batchExtras,
     Universal / mappings ++= Seq(
-      file("serv/src/omega-edit-grpc-server.js") -> "omega-edit-grpc-server.js"
+      file("serv/src/omega-edit-grpc-server.js") -> "bin/omega-edit-grpc-server.js"
```

Comment:
@scholarsmate Since you change the directory of `omega-edit-grpc-server.js` you need to update `src/rpc/client/ts/src/server.ts` to run `bin/omega-edit-grpc-server.js` instead of `omega-edit-grpc-server.js`